readthedocs/core/signals.py

```diff
@@ -62,4 +67,23 @@ def decide_if_cors(sender, request, **kwargs):  # pylint: disable=unused-argumen
 
     return False
 
+
+@receiver(pre_delete, sender=User)
+def delete_projects_and_organizations(instance, *args, **kwargs):
+    # Here we count the owner list from the projects that the user own
+    # Then exclude the projects where there are more than one owner
+    projects_id = (instance.projects.all().annotate(num_users=Count('users'))
+                                          .exclude(num_users__gt=1)
+                                          .values_list('id', flat=True))
```

Comment:
I'll have to think more on this logic, but at first glance this will cause problems with our hosting at readthedocs.com, as users don't own projects, a more top-level object, `Organizations` do. Projects shouldn't be automatically removed like this in that case.